tensorboard/webapp/tbdev_upload/tbdev_upload_dialog_component.css

```diff
@@ -0,0 +1,46 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+:host > div,
```

Comment:
you do not need to scope it like this: `div` is sufficient. CSS rules are all encapsulated (Angular stamps DOM with random property and these CSS rules get prefixed/modified to contain that hash at the build time).

Reply:
I was inspired by what I found here:
https://github.com/tensorflow/tensorboard/blob/master/tensorboard/webapp/settings/dialog_component.css

Right now there aren't other divs but most likely there will be.  Happy to remove the ":host >" for now but something like it will evenutally need to return.